modeltree.h

```diff
@@ -36,38 +36,50 @@
 namespace Asn1Acn {
 namespace Internal {
 
+class ProjectContentHandler;
+
 class ModelTree : public QObject
 {
     Q_OBJECT
 
     ModelTree();
     ~ModelTree() = default;
 
+    friend class ProjectContentHandler;
```

Comment:
is that `friend` necessary? (it usually leads to a little bit messier architecture